WalletWasabi.Fluent/Converters/SearchIconConverter.cs

```diff
@@ -0,0 +1,35 @@
+using Avalonia;
+using Avalonia.Data.Converters;
+using System;
+using System.Globalization;
+using WalletWasabi.Fluent.ViewModels.Search;
+
+namespace WalletWasabi.Fluent.Converters
+{
+	public class SearchIconConverter : IValueConverter
```

Comment:
Why couldnt we use the NavBarIconConverter here?

If its not possible we can probably have a shared baseclass?

Reply:
Adding converter seems like simpler solution than adding additional layer of inheritance, also considered adding interface but decided to add new converter.

Reply:
Refactored code so SearchItemViewModel inherits directly from NavBarItemViewModel